stylix/nixos/cursor.nix

```
@@ -0,0 +1,10 @@
+{ config, ... }:
+
+let
+  cfg = config.stylix.cursor;
+in {
+  imports = [ ../cursor.nix ];
+  config = {
+    environment.variables.XCURSOR_SIZE = "${toString cfg.size}";
```

The `"${...}"` was converting a string to a string.

```suggestion
    environment.variables.XCURSOR_SIZE = toString cfg.size;
```